packages/experimental/Text/src/Text.tsx

```diff
@@ -51,14 +51,12 @@ export const Text = compressible<TextProps, TextTokens>((props: TextProps, useTo
     ? 'right'
     : align;
 
-  const onAccTap =
-    onAccessibilityTap ??
-    React.useCallback(
-      (event?) => {
-        props.onPress(event);
-      },
-      [props.onPress],
-    );
+  const onAccTap = React.useCallback(
+    (event?) => {
+      onAccessibilityTap ? onAccessibilityTap() : props.onPress(event);
+    },
+    [props.onPress],
```

Comment:
`onAccessibilityTab` should also be in the dependency array? Also I thought we had an enlist rule that told us not to do`foo.bar` inside the array and instead pull out `bar` as a separate var. AKA, pull out onPress between lines 21-36 so that we can just have `onPress` instead of `props.onPress` in the array.